bundles/af-core/pom.xml

```diff
@@ -459,6 +459,7 @@
         <dependency>
             <groupId>com.adobe.aem</groupId>
             <artifactId>aem-forms-sdk-api</artifactId>
+            <version>2024.04.18.00-240300</version>
```

Comment:
This change should be in the parent pom, version should not be present in individual modules.